SystemInfo/Exception/ComposerJsonFileNotFoundException.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+/**
+ * File containing the ComposerJsonFileNotFoundException class.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\EzSupportToolsBundle\SystemInfo\Exception;
+
+use Exception;
+use eZ\Publish\Core\Base\Exceptions\NotFoundException as BaseNotFoundException;
+
+class ComposerJsonFileNotFoundException extends BaseNotFoundException
+{
+    public function __construct($path, Exception $previous = null)
```

Comment:
```suggestion
    public function __construct(string $path, Exception $previous = null)
```
if path is `string` we can add type hint